### DISCUSSION TITLE
iptables issue when using Rocky Linux 9 (RHEL 9)

### DISCUSSION THREAD

**Discussion by alwism** [wg-easy/wg-easy#975] [Q&A]
**Describe the bug**
Container doesn't stay up and loops the same error log provided in the screenshot

**To Reproduce**
1. Install Rocky Linux 9
2. Statting the container with the default docker-compose.yml attached

**Expected behavior**
Container does stay up

**Screenshots**
![grafik](https://github.com/wg-easy/wg-easy/assets/60648387/1bbe52d2-603a-4411-82cc-f20130110f78)

**Server:**
 - OS: Rocky Linux 9
 - Kernel: 5.14.0-362.24.1.el9_3.x86_64
 - Docker: Docker version 26.0.0, build 2ae903e

**Additional context**
Found this releated issue:
https://ryandaniels.ca/blog/docker-and-trouble-with-red-hat-enterprise-linux-9-iptables/

**Comment by pheiduck** [comment]
This is not really an issue of wg-easy as you found out.
Some users solved this by modprobe iptables as the article say.

**Comment by alwism** [comment]
I know but some documentation or usefull error would be great :D
